src/Actions/babylon.actionManager.ts

```diff
@@ -62,16 +62,17 @@
         private static _OnRightPickTrigger = 3;
         private static _OnCenterPickTrigger = 4;
         private static _OnPickDownTrigger = 5;
-        private static _OnPickUpTrigger = 6;
-        private static _OnLongPressTrigger = 7;
-        private static _OnPointerOverTrigger = 8;
-        private static _OnPointerOutTrigger = 9;
-        private static _OnEveryFrameTrigger = 10;
-        private static _OnIntersectionEnterTrigger = 11;
-        private static _OnIntersectionExitTrigger = 12;
-        private static _OnKeyDownTrigger = 13;
-        private static _OnKeyUpTrigger = 14;
-        private static _OnPickOutTrigger = 15;
+        private static _OnDoublePickTrigger = 6;

```

Comment:
I did this just to ensure that the hasPickTriggers and hasPointerPickTriggers are still working without changing the code. You check the range of action.trigger value which must be greater than the lowest pick trigger (onPickTrigger = 1) and lower than the greatest (onPickUpTrigger = 6 for hasPickTriggers and onPointerOutTrigger = 9 for hasPointerTriggers). So I inserted the double click trigger value inside these two ranges in order to prevent any changes for other functions.
What you want me to do ? Change the double click trigger value (16 for example),  and complete conditions in hasPickTriggers and hasPointerTriggers ? Keep my changes  because the values of triggers are not supposed to be changed and seen ?